website/docs/r/cloudsubnet.html.markdown

```diff
@@ -21,14 +22,17 @@ Manages ACI Cloud Subnet
 		scope              = "public"
 		usage              = "user"
 	}
+
 ```
+
+
 ## Argument Reference ##
 * `cloud_cidr_pool_dn` - (Required) Distinguished name of parent CloudCIDRPool object.
 * `ip` - (Required) CIDR block of Object cloud_subnet.
 * `annotation` - (Optional) annotation for object cloud_subnet.
 * `name_alias` - (Optional) name_alias for object cloud_subnet.
 * `scope` - (Optional) The domain applicable to the capability. Allowed values are "public", "private" and "shared". Default is "private".
-* `usage` - (Optional) The usage of the port. This property shows how the port is used. Allowed values are "user" and "infra-router". Default is "user".	
+* `usage` - (Optional) The usage of the port. This property shows how the port is used. Allowed values are "user", "gateway" and "infra-router". Default is "user". If `hub_network` is set for `aci_cloud_context_profile`, then value of `usage` must be "gateway".	
```

Comment:
Let's change the last if statement to "To make any subnet a Gateway subnet use usage = gateway."